### ISSUE TITLE
Does not work

### ISSUE THREAD

**Issue by StarpTech** [brunch/postcss-brunch#52]
I get no issues but the generated styles aren't prefixed.

app.scss
```scss
.brunch {
  font-family: -apple-system, Sans-Serif;
  text-align: center;
  font-size: 24pt;
  color: #3f894a;
}

a {
  display: flex;
  transition: all .5s;
  user-select: none;
  background: linear-gradient(to bottom, white, black);
}
```
brunch-config.js
```js
// See http://brunch.io for documentation.
exports.files = {
  javascripts: {
    joinTo: {
      "vendor.js": /^(?!app)/, // Files that are not in `app` dir.
      "app.js": /^app/
    }
  },
  stylesheets: { joinTo: "app.css" }
};


exports.plugins = {
  postcss: {
    processors: require('autoprefixer')(['last 4 versions'])
  }
};

```
package.json
```json
{
  "name": "brunch-app",
  "description": "Brunch.io application",
  "private": true,
  "author": "Brunch",
  "version": "0.0.1",
  "repository": "",
  "scripts": {
    "start": "brunch watch --server",
    "build": "brunch build --production"
  },
  "dependencies": {
    "font-awesome": "^4.7.0",
    "jquery": "^3.2.1"
  },
  "devDependencies": {
    "auto-reload-brunch": "^2",
    "autoprefixer": "^7.1.6",
    "babel-brunch": "~6.0",
    "babel-preset-env": "^1.6.1",
    "brunch": "^2",
    "clean-css-brunch": "^2",
    "csswring": "^6.0.2",
    "node-sass": "^4.7.2",
    "postcss": "^6.0.14",
    "postcss-brunch": "^2.1.0",
    "sass-brunch": "^2.10.4",
    "uglify-js-brunch": "^2.10.0"
  }
}

```

**Output:**

```css
.brunch {
  font-family: -apple-system, Sans-Serif;
  text-align: center;
  font-size: 24pt;
  color: #3f894a; }

a {
  display: flex;
  transition: all .5s;
  user-select: none;
  background: linear-gradient(to bottom, white, black); }


/*# sourceMappingURL=app.css.map*/
```

**Comment by shotleybuilder** [comment]
Yes, same here.  I've installed into a Elixir Phoenix project with almost the same package.json.  No errors but no output for the autoprefixer plugin.  What versions of sass-brunch, brunch and postcss play well together?

**Comment by phonezawphyo** [comment]
i use postcss-brunch ~0.5.0 for phoenix

**Comment by kgcreative** [comment]
https://github.com/helols/spostcss-brunch works for me with the latest version of brunch.

tl;dr, postcss-brunch runs before sass-brunch because of alpha order in package.json. spostcss-brunch tracks post-css brunch, but loads after sass-brunch because alpha order, which restores the proper chaining order and prefixes all my css correctly.
